builds/azure-pipelines/build-release-java.yml

```diff
@@ -0,0 +1,30 @@
+steps:
+  - task: Maven@3
+    displayName: Build library
+    inputs:
+      mavenPomFile: java-library/pom.xml
+      options: --batch-mode --define=gpg.skip --define=org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --update-snapshots
+
+  - powershell: |
+      $prefix = 'azure-functions-java-library-sql-0.1.0'
```

Comment:
Instead of hardcoding the version here what about just using the copy task with star patterns?

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/copy-files?view=azure-devops&tabs=yaml

(and I don't think you need to make a version-named target folder for this. Just name it something generic like java-library or something)